.env.sample

```diff
@@ -11,4 +11,5 @@ FIREBASE_DYNAMIC_LINK_URL=
 FIREBASE_API_KEY=
 BETTER_WEB_APP_URL=
 BETTER_APP_STORE_ID=
-DATABASE_URL=
\ No newline at end of file
+DATABASE_URL=
+SEGMENT_WRITE_KEY=
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [14-16]

The new environment variable `SEGMENT_WRITE_KEY` is added correctly. However, it's a good practice to add a newline at the end of the file to avoid any potential issues with programs that expect or require it (like Unix-based tools).
```diff
- \ No newline at end of file
+ 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->